### DISCUSSION TITLE
Different thresholds for same rule

### DISCUSSION THREAD

**Discussion by damienmiheev** [detekt/detekt#4055] [Q&A] [answered]
I'm using Detekt and I need to configure LargeClass rule. I want to set different thresholds for production and test classes. Is it possible to configure it such way?

**Comment by cortinico** [comment, marked as answer]
> Is it possible to configure it such way?

I believe not. The closest approximation to this is to enable the rule for production, and filter out your test classes with an `exclude`.

You could also create your own `LargeTestClass` rule that delegates to `LargeClass`. This will be a fully separated rule so you can configure it separately (and specify also a different exclude).
